articles/cloud-storage/cs-how-use-file-browsers.md

```diff
@@ -146,78 +146,64 @@ To install and configure S3 Browser:
     > [!NOTE]
     > Not all S3 features are implemented by the ECS S3 compatible API, so some functionality may not work. However, common file operations (create, read, write and delete) all work well.
 
-## Browser extensions
+## Internet Browser Access
 
-This section takes you through installing and configuring browser extensions that enable you to browse the contents of Cloud Storage using your web browser.
+This section takes you through installing and configuring browser access that enables users to browse the contents of Cloud Storage using their web browser.
 
-Atmos extensions are available for both Chrome and Firefox. Although the extensions included in this article were natively built for the Atmos platform, they also work with ECS using Atmos endpoints and therefore work well with Cloud Storage.
+An EMC created HTML5 based application can be installed to your Atmos bucket, which generates a URL that enables customers to share with users who require access to cloud storage.
 
-S3 extensions are also available for both browsers, but none of them include the functionality to customise the server host name, which is a requirement for connecting to Cloud Storage. If we find S3 browser extensions that offer this functionality in the future, we'll update this article.
+   > [!NOTE]
+   > If you have multiple buckets under the same endpoint these can all be accessed from a single URL, however if you have multiple buckets hosted in both Cloud Storage regions, you will need to install the application on both regions and access the relevent buckets using the relevent URL.
 
-### Chrome — Atmos for Chrome
+S3 extensions are also available for browsers, but none of them include the functionality to customise the server host name, which is a requirement for connecting to Cloud Storage. If we find S3 browser extensions that offer this functionality in the future, we'll update this article.
 
-To install and configure Atmos for Chrome:
+### Atmos HTML5 Application
 
-1. Download and install the [Atmos for Chrome extension](https://chrome.google.com/webstore/detail/atmos-for-chrome/befofpjhdclgccimklkaheifjajbcmnk).
+   > [!NOTE]
+   > For installation and initial configuration of the Atmos HTML5 application, you will need the following Prerequisite information:
+   > Atmos Access Key - in the form of SubtennantID/User
```

Comment:
I think you need to put an empty line above this one (and the following two) for this to render correctly. Although the empty lines will need to begin with the > so that the note remains intact